src/main/scala/com/microsoft/azure/cosmosdb/spark/rdd/CosmosDBRDDIterator.scala

```diff
@@ -366,6 +366,7 @@ class CosmosDBRDDIterator(hadoopConfig: mutable.Map[String, String],
           updateTokens(currentToken, nextToken, partitionId)
 
           logDebug(s"changeFeedOptions.partitionKeyRangeId = ${changeFeedOptions.getPartitionKeyRangeId}, continuation = $currentToken, new token = ${response._2}, iterator.hasNext = ${response._1.hasNext}")
+          System.out.println(s"changeFeedOptions.partitionKeyRangeId = ${changeFeedOptions.getPartitionKeyRangeId}, continuation = $currentToken, new token = ${response._2}, iterator.hasNext = ${response._1.hasNext}")
```

Comment:
can you remove the print statement and use the logger instead ?

Reply:
Left-over from debugging. Thanks for catching this